config/finders/services.yml

```diff
@@ -0,0 +1,63 @@
+---
+analytics_identifier:
+base_path: "/services"
+content_id: f6d779ac-5f78-413d-a1ff-da391944e6ec
+content_purpose_document_supertype: navigation
```

Comment:
I think this supertype and the others aren't used. Can we remove them if that's the case? Might be worth looking at the other entries as well.

Reply:
Thanks, @tijmenb! You're right - I removed them.